themes/upbound/assets/scss/_color-mode-light.scss

```diff
@@ -4,6 +4,7 @@
     // body and content
     --body-background: white;
     --body-font-color: #030724;
+    --body-selection: #EBDFFE66;
```

Comment:
Note: Add an opacity level to the color so that the selection background color works over dark backgrounds as well like code blocks